FreeRTOS/FreeRTOS
# Add SMP disable preemption on target test

Add SMP disable preemption on target test

Description
-----------
Adding the on target test to verify that
> The scheduler shall not schedule a task that is pinned to a specific core on any other core.

Test Steps
-----------
Test result
```
./FreeRTOS/FreeRTOS/Test/Target/tests/smp/disable_preemption/disable_preemption.c:232:Test_DisablePreemption:PASS

-----------------------
1 Tests 0 Failures 0 Ignored
OK
```

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
